### ISSUE TITLE
Page number not completely visible on some devices

### ISSUE THREAD

**Issue by OdisBy** [mihonapp/mihon#1105]
### Steps to reproduce

1. Install the device on devices that have some type of small edge (maybe only from Samsung?)
2. Have the Show page number option enabled in general reading settings
3. Open some chapter

### Expected behavior

Being able to fully see the page number at the bottom of screen as the image below
<details><summary>Screenshot from my device with open chapter</summary>
<p>

![image](https://github.com/user-attachments/assets/6c97bb1d-562d-4647-bd71-f5ab2c6ccabe)

</p>
</details> 

### Actual behavior

On the real device the page number is cut off, and it is not possible to read the numbers. It is hidden by the small edges that the device has. Both in the screenshot and in the emulator the number will appear correctly, so I will attach a photo of my real device with this problem.
<details><summary>Full Picture</summary>
<p>

![image](https://github.com/user-attachments/assets/a2855082-dd44-46e4-8b09-4760f90b627d)

</p>
</details> 

<details><summary>With zoom on page numbers</summary>
<p>

![image](https://github.com/user-attachments/assets/9a7ca708-4fb5-4537-8f3b-4b0560639eb4)

</p>
</details> 

### Mihon version

0.16.5

### Android version

Android 14

### Device

Samsung A54

### Other details

This issue seems to be related to the edges of the devices. I noticed that when taking a screenshot or mirroring my phone, it appears correctly at the bottom of the screen, but this is not the visible behavior on the actual device. For the same reason, emulating a device will display it correctly.

It won't be possible to fully demonstrate this behavior with screenshots from the devices, so I have attached a photo of my actual device showing this problem in the Actual Behavior section. Additionally, I will attach some related complaints that I found from other users both on Discord and in an Issue here on GitHub.

Issue on Github #526

<details><summary>Message on Support channel (Mihon Discord) </summary>
<p>

[https://discord.com/channels/1195734228319617024/1196111629482463422/1198382299763445871](https://discord.com/channels/1195734228319617024/1196111629482463422/1198382299763445871)

![image](https://github.com/user-attachments/assets/5182ef86-cfa4-4935-a0e4-62b0a8f3b269)

</p>
</details> 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by OdisBy** [comment]
I'd like to work on this issue

**Comment by AntsyLich** [comment]
Not a Mihon issue since it's fine on the screenshot. Your device has defect.

**Comment by OdisBy** [comment]
I agree, it seems to be an issue with the phone's frame. Although it's not a problem with Mihon, can I add a way to allow for this padding? Maybe I could add a configuration option for a bottom margin, and those who need it could enable it? I could set it to something like 4 dp of margin, or perhaps offer a configuration where the user can choose this padding? In that case, it would be available in the Settings.

**Comment by FooIbar** [comment]
> Not a Mihon issue

Technically It is.
Mihon didn't handle display cutout insets.

@OdisBy You can disable "show content in cutout area" as a workaround before a proper fix is landed.

**Comment by AntsyLich** [comment]
Ah that makes sense. Are you working on a fix or I can go ahead?

**Comment by FooIbar** [comment]
> Ah that makes sense. Are you working on a fix or I can go ahead?

I don't have time now. I can do it later in the week if it's not fixed yet.

**Comment by OdisBy** [comment]
I was planning to work on this bug, but since it could impact all users and this would be my first contribution, I prefer to leave it for the AntsyLich and take on something else!

**Comment by AntsyLich** [comment]
@OdisBy does this issue occur in any other place in the app?

**Comment by OdisBy** [comment]
@AntsyLich I just tested the entire application, I didn't find anywhere else with this problem
